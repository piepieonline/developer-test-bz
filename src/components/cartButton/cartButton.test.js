import React from 'react';
import ReactDOM from 'react-dom';
import CartButton from './cartButton';

it('renders the expected HTML', () => {
    const div = document.createElement('div');
    ReactDOM.render(<CartButton />, div);

    expect(div).toMatchSnapshot();

    ReactDOM.unmountComponentAtNode(div);
});