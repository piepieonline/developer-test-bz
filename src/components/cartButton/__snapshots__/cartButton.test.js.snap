// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the expected HTML 1`] = `
<div>
  <div
    class="cart-button-container"
  >
    <input
      class="cart-button"
      type="button"
      value="Add to Cart"
    />
    <div
      class="cart-button-status-popup"
      style="opacity: 0;"
    >
      <span
        class="cart-button-status"
      >
        Item added to cart!
      </span>
    </div>
  </div>
</div>
`;
